pages/guides/drop-ins/generate-client-token.mdx

```diff
@@ -58,3 +57,70 @@ function generateClientTokenWithBody(body) {
     });
 }
 ```
+
+##### Example configuration using `tokenUrl`
+
+```javascript
+<script>
+  dwolla.configure({
+      environment: "sandbox",
+      // styles: "/styles/create-custom.css",
+      success: (res) => Promise.resolve(),
+      error: (err) => Promise.resolve(),
+      tokenUrl: "/tokenUrl",
+  });
+</script>
+```
+
+### Using Token
+
+Create a function that calls your server-side `tokenUrl` endpoint and returns a promise containing a token. Then set up the configuration of the drop-in with a `token` parameter which calls that function. This allows passing custom headers when calling your server-side endpoint for added protection and authentication.
```

Comment:
Smol suggested edits:

Using the `token` option when configuring the dwolla-web.js library allows you to create a function that calls your server-side `tokenUrl` endpoint (As described above). The function you create takes in two arguments, 1) the request body that is required to make the /client-tokens API request; and 2) a JSON object the includes key:value pairs for specifying custom headers. When an action needs to be performed within a flow component, the dwolla-web.js library will call your custom function and dynamically pass in a request body as the first argument. The server-side endpoint should act as a pass-through by taking in the request body required in order to make the appropriate call to the /client-tokens endpoint. Upon success, the /client-tokens endpoint will return response body including a "token" string value. The HTTP response that is returned to the client-side is expecting an object in the format of `{token: 'Token string value'}`.